templates/colors/literals-swift4.stencil

```diff
@@ -1,5 +1,7 @@
 // Generated using SwiftGen, by O.Halligon — https://github.com/SwiftGen/SwiftGen
 
+// swiftlint:disable all
```

Comment:
This might be better  above the `// Generated using SwiftGen` comment because if you have a SwiftLint rule about your headers then it trips up the linter.  Though I've not tested that.  I had to just add the generated files to our SwiftLint exclusion list.

Reply:
Oh yeah, that makes perfect sense. Should have thought of that previously...